src/actinia_statistic_plugin/endpoints.py

```diff
@@ -35,6 +35,14 @@
 # from .raster_sampling_geojson import AsyncEphemeralRasterSamplingGeoJSONResource, \
 #    SyncEphemeralRasterSamplingGeoJSONResource
 
+from .vector_sampling import (
+    AsyncEphemeralVectorSamplingResource,
+    SyncEphemeralVectorSamplingResource,
+)
+
+# from .vector_sampling_geojson import AsyncEphemeralVectorSamplingGeoJSONResource, \
+#    SyncEphemeralVectorSamplingGeoJSONResource
```

Comment:
I'd prefer to remove commented code.

Reply:
done in 680b667d5a051f3235c500c635d9b1ba1dd5d617